src/components/Column/ColumnHeader.tsx

```diff
@@ -6,12 +6,13 @@ import IconButton from "../IconButton";
 
 interface ColumnHeader {
   title: string;
+  onTitleChange: (value: string) => void;
```

Comment:
[nitpick] - Does this need to be this name? It feels like this name indicates that it gets triggered every time when you type something on the text input because `onTitleChange` doesn't really tell me when exactly it's expected to invoke the passed property. Something like `titleChangeOnBlur` came to my mind but you might have a better idea.

Reply:
BTW - it's great this property is blindly optional 😄